src/tests/multihost/sssd/testlib/common/utils.py

```diff
@@ -83,7 +83,7 @@ def client_install_pkgs(self):
                'tcpdump wireshark-cli expect rsyslog gcc gcc-c++ pam-devel '\
                'tdb-tools libkcapi-hmaccalc strace'
         sssd_pkgs = 'sssd sssd-tools sssd-proxy sssd-winbind-idmap '\
-                    'libsss_autofs sssd-kcm sssd-dbus'
+                    'libsss_autofs sssd-kcm sssd-dbus iproute-tc'
```

Comment:
Should be probably in pkgs as it is not a sssd one.

Reply:
updated.